features/viewpoint_deletion.feature

```diff
@@ -0,0 +1,12 @@
+#language: fr
+
+Fonctionnalité: Supprimer un point de vue (#60 Delete a viewpoint)
```

Comment:
While it might be a good idea to give the issue number and the title in English, either you do it for ALL scenarios or for none.